user-guide/modules/ROOT/pages/advanced-builds.adoc

```diff
@@ -1,3 +1,247 @@
++
+[source,txt]
+----
+./b2 link=static
+----
++
+Apart from distributing applications, the other compelling reasons to use static linking are to avoid dependency or version conflicts, and perhaps improved performance, as the application doesn't need to load shared libraries at runtime.
+
+. The `link=dynamic` option specifies dynamic linking, also known as shared linking. With dynamic linking, the linker only includes a small amount of code to locate and interface with the actual library code, which is kept in a separate file. This results in smaller executables and allows for library code to be shared between multiple programs. However, the resulting program requires that the shared library be available at both build time and run time. The build options `dynamic` and `shared` are synonymous.
++
+[source,txt]
+----
+./b2 link=dynamic
+./b2 link=shared
+----
+
+Not all Boost libraries need to be linked to your program. Many are "header-only", meaning you only need to include their headers in your source files and don't need to worry about linking, as there is nothing to link. 
+
+== Build Boost with CMake
+
+The only supported method of building the Boost libraries is with the `b2` tool. However, there are experimental alternatives, including building with CMake. Refer to the https://github.com/boostorg/cmake[Boost CMake support infrastructure] for full details, including information on general and library-specific configuration variables, testing after building with CMake, using Boost (or an individual Boost library) with the `add_subdirectory` command, and using Boost with the `FetchContent` command.
+
+== Parallel Builds
+
+To speed up the build process, you might want to utilize parallel compilation. This is where multiple compilation tasks are performed concurrently to speed up the build process. You can enable this feature by using the `-j` option followed by the number of tasks you want to run in parallel when invoking `b2`.
+
+For example, let's say you have a Jamroot.jam file at the root of your project that describes how to build your project. The project consists of several source files spread across different directories, and you are working on a multi-core machine where parallel builds could significantly speed up the build time. To build your project using Boost.Build, you would normally navigate to your project's root directory and run `b2`. However, to take advantage of parallel builds, you can specify the `-j` option followed by the number of tasks you want to run concurrently. If you have a quad-core processor, you might run up to four tasks in parallel:
+
+[source,txt]
+----
+b2 -j4
```

Comment:
I imagined we were changing the top level topic from `b2` to `cmake` here. If the intention was `cmake` instead of `b2`, then `cmake -j4` is not a valid command. If we moved the higher level topic from `b2` to `cmake` for just one paragraph and moved back to `b2` without listing any cmake build options, then the previous section is completely out of place.